### PR TITLE
[CS-1621] - Adds max button and gas fee on send tokens from depot

### PR DESCRIPTION
**Pull request by danibonilha**
### Description

Adds max button and gas fee on send tokens from depot, 
Extracted some rainbow's logic in order to reuse the component, the commit are pretty self explanatory, the plan is to add some tests when we get some time, hopefully I didn't break anything on both flows, will be adding a .gif on monday, just to have more time to validate all the scenarios.

- [x] Completes #(CS-1621)

### Screenshots

send-max

### CODE REVIEW

File: cardstack/src/components/Button/Button.tsx [hunk 1]
```diff
@@ -119,8 +116,7 @@ export const Button = ({
           position="absolute"
           height="100%"
           zIndex={1}
-          width={width}
-          maxWidth={maxWidth}
+          width="100%"
```
Comment (topwebtek7) [inline cardstack/src/components/Button/Button.tsx:119]: @danibonilha did you confirm if this change does not break any existing buttons?

**Review by topwebtek7 — APPROVED** [review]
Looks good to me. just left a comment
And will you add tests for max button behavior and sending tokens from depot?

**Comment by eshinkawa** [comment]
LGTM

**Review by eshinkawa — APPROVED** [review]

File: cardstack/src/components/Button/Button.tsx (hunk 1 above)
Comment (danibonilha) [reply cardstack/src/components/Button/Button.tsx:119]: Yes, on the buttons I could find, I didn't find any other disabled button, so I'm guessing this change only affects this flow

**Comment by danibonilha** [comment]
> Looks good to me. just left a comment
> And will you add tests for max button behavior and sending tokens from depot?

Will be adding tests, once merchant flow is complete

**Pull request merged**